packages/playground/src/views/nodes.vue

```diff
@@ -528,8 +528,8 @@ export default {
             domain: filters.value.gateway || undefined,
             freeIps: +filters.value.publicIPs || undefined,
             dedicated: filters.value.dedicated || undefined,
-            sortBy: SortBy.Status,
-            sortOrder: SortOrder.Asc,
+            sortBy: SortBy.FreeCRU,
+            sortOrder: SortOrder.Desc,
```

Comment:
It was `Asc` why we changed it to `Desc`

Reply:
```Asc``` takes the nodes with small free cores in the first etc ```node with ID 11``` which we don't need.